calculatorTest/LoginPageTest.cs

```diff
@@ -8,16 +9,13 @@ namespace calculatorTest
 {
     public class LoginPageTest
     {
-
-        const string BASEURL = "http://localhost:64177/";
         IWebDriver browser;
 
         [SetUp]
         public void Setup()
         {
             browser = new ChromeDriver();
-            browser.Url = (BASEURL + "Login");
-
+            browser.Url = (GeneralSettings.BASEURL + "Login");
```

Comment:
Not an issue, just for your information: you don't need to take it in brackets.

```suggestion
            browser.Url = GeneralSettings.BASEURL + "Login";
```